### ISSUE TITLE
repositories added with apt::ppa are purged by apt

### ISSUE THREAD

**Issue by maxadamo** [puppetlabs/puppetlabs-apt#1070]
## Describe the Bug

I have set these options:

```puppet
class { 'apt':
  purge  => {
    'sources.list'   => true,
    'sources.list.d' => true,
  }
}
```

then I add: 

```puppet
apt::ppa { 'ppa:deadsnakes/ppa': }
```

the repo name is taken here: [deadsnakes-ppa](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa)

and every time Puppet runs the repo is added and then it's removed, again and again. 

## Expected Behavior

it should not purge the repository added through apt::ppa

## Steps to Reproduce

see above

## Environment

 - puppetlabs/apt version: 9.0.0
 - puppet agent and server version: 7.xx
 - Platform: Ubuntu 20.04

## Additional Context

N/A

**Comment by kenyon** [comment]
Duplicate of #1064.

**Comment by maxadamo** [comment]
solved by #1069
